srcpkgs/libsearpc/patches/python3.patch

```diff
@@ -0,0 +1,20 @@
+--- pysearpc/server.py.ORIG	2018-06-21 04:49:39.000000000 +0200
++++ pysearpc/server.py	2018-07-10 08:44:54.404532828 +0200
```

Comment:
The patch is incomplete. Here's a complete one: https://gist.github.com/asergi/a6c521ff888eca0038d2d7266383efbc